src/styles/theme.ts

```diff
@@ -0,0 +1,201 @@
+const white = '#FFF'
+const blue = '#218DFF'
+const blueDark = '#0b66c6'
+const bgLight = '#edf2f7'
+const bgDark = '#2e2e2e'
+const blackLight = '#181a1b'
+// const black = '#000'
+
+export enum ThemeType {
+  MAIN = 'main',
+  SECONDARY = 'secondary',
+}
+
+interface ColourThemeBase {
+  [t: string]:
+    | {
+        [key in ThemeType]?:
+          | {
+              [prop: string]: string
+            }
+          | string
+          | undefined
+      }
+    | string
+}
+
+export type ColourTheme = ColourThemeBase & {
+  bg1: string
+}
+
+const darkTheme: ColourTheme = {
+  bg1: bgDark,
+  bg2: blackLight,
+  button: {
+    main: {
+      text1: blue,
+      text1Hover: white,
+      get bg1(): string {
+        return darkTheme.bg1
+      },
+      bg1Hover: blueDark,
+    },
+    secondary: {
```

Comment:
`main and secondary` end up the same, don't they?

Reply:
wdym?

Reply:
ignore me, I was looking at something different